omd/packages/nagios/skel/local/lib/nagios/plugins/check_crl

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright (C) 2013 - Remy van Elst
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Mark Ruys <mark.ruys@peercode.nl> - 2015-8-27
+# Changelog: - catch openssl parsing errors
+#            - clean up temporary file on error
+#            - add support for PEM CRL's
+#            - fix message when CRL has been expired
+#            - pretty print duration
+
+# Jeroen Nijhof <jnijhof@digidentity.eu>
+# Changelog: - fixed timezone bug by comparing GMT with GMT
+#            - changed hours to minutes for even more precision
+
+# Remy van Elst - raymii.org - 2012
+# 05.11.2012
+# Changelog: - check with hours instead of dates for more precision,
+#            - URL errors are now also catched as nagios exit code.
+
+# Michele Baldessari - Leitner Technologies - 2011
+# 23.08.2011
+
+# Oguzhan Cicek - OpenSource Security GmbH - oguzhan(at)os-s.de - 2021
+# 05.04.2021
+# Changelog: - added argparse parsing
+#            - added handling for DER-Certificates
+
+import time
+import datetime
+import getopt
+import os
+import pprint
+import subprocess
+import sys
+import tempfile
+import typing
+import argparse
+import urllib.request, urllib.parse, urllib.error
+
+def output_check_result(rc, s):
+    stxt = ['OK', 'WARN', 'CRIT', 'UNKNOWN'][rc]
```

Comment:
I don't think you need `stxt`, since in Checkmk, you anyway see the check result in a separate column, so there is no need to add it here as a text.